migrations/20200716100000-set-site-subdomain.js

```diff
@@ -0,0 +1,17 @@
+const yaml = require('js-yaml');
+const { generateSubdomain } = require('../api/utils');
+
+const cmdUpdateDedicatedBucketSitesSubdomain = 'UPDATE site SET subdomain = "s3ServiceName" WHERE "s3ServiceName" !~ \'federalist-(production|staging)-s3\'';
+const getSharedBucketSites = 'SELECT id, owner, repository from site WHERE "s3ServiceName" ~ \'federalist-(production|staging)-s3\'';
+const cmdUpdateSiteSubdomain = site => `UPDATE site SET subdomain = '${generateSubdomain(site.owner, site.repository)}' WHERE id = ${site.id}`;
+const cmdUnsetAllSiteSubdomains = 'UPDATE site SET subdomain = null';
+
+exports.up = (db, callback) => db.runSql(cmdUpdateDedicatedBucketSitesSubdomain)
```

Comment:
Why do we need to 2 different update statements, can we just update all the sites to the result of `generateSubdomain`? It shouldn't matter what the existing value is, right?